themes/default/layouts/page/leadership.html

```diff
@@ -0,0 +1,56 @@
+{{ define "hero" }}
+    {{ partial "hero" (dict "title" .Params.title ) }}
+{{ end }}
+
+{{ define "main" }}
+    <div class="container mx-auto mt-12  max-w-7xl">
+        <a class="text-blue-600 text-xl font-body ml-9 hidden sm:block" href="{{ relref . " /about" }}">
+            <i class="fas fa-arrow-left mr-4"></i>
+            About
+        </a>
+
+        <section id="leaders" class="mb-28">
+            <div class="container mx-auto text-center">
+                <h2 class="mb-8 text-center">Our Leaders</h2>
+                <ul class="inline-flex flex-wrap list-none justify-center p-0 gap-x-8 gap-y-6">
+                    {{ range (sort $.Site.Data.team.leadership "weight") }}
+                        <li class="about-card border-2 border-gray-300 rounded-md flex overflow-hidden relative">
+                            <div class="flex">
+                                <img class="" src="/images/team/{{ .id }}.jpg" alt="{{ .name }}" />
+                                <div class="flex flex-col justify-center text-left mx-4">
+                                    <div class="font-display font-medium text-gray-900 text-3xl mb-3">{{ .name }}</div>
+                                    <div class="text-sm text-gray-700">{{ .title }}</div>
+                                </div>
+                                <div class="opacity-0 hover:opacity-95 bg-blue-600 h-full w-full absolute flex justify-center social-overlay">
+                                    {{ partial "widgets/social-icons.html" (dict "social" .social) }}
+                                </div>
+                            </div>
+                        </li>
+                    {{ end }}
+                </ul>
+            </div>
+        </section>
+
+        <section id="board" class="mb-28">
+            <div class="container mx-auto text-center">
+                <h2 class="mb-8 text-center">Our Board of Directors</h2>
```

Comment:
please add the name of the firms for each board member

Reply:
@gthuang Where can I get that information? It's not on our current about page.

Reply:
our investors are madrona, tola capital, and NEA. you can check their linkedin pages to confirm who works where

Reply:
Okay, I just want to be really confident that I'm using the right names for the right companies, and that all the linked in pages are up to date - if LinkedIn will be a 100% accurate source of truth, I will look there.